eclipse-scout-core/src/form/fields/wrappedform/WrappedFormField.ts

```diff
@@ -65,21 +74,16 @@ export class WrappedFormField extends FormField implements WrappedFormFieldModel
       return;
     }
 
-    this.innerForm.setDisplayHint(Form.DisplayHint.VIEW); // by definition, an inner form is a view.
-    this.innerForm.setModal(false); // by definition, an inner form is not modal.
-    this.innerForm.setClosable(false); // Disable close key stroke
-    this.innerForm.renderInitialFocusEnabled = this.initialFocusEnabled; // do not render initial focus of form if disabled.
-    if (this.innerForm.rootGroupBox) {
-      this.innerForm.rootGroupBox.setMenuBarPosition(GroupBox.MenuBarPosition.BOTTOM);
-    }
-
+    this.innerForm.renderInitialFocusEnabled = this.initialFocusEnabled;
     this.innerForm.render();
 
     this.addField(this.innerForm.$container);
     this.innerForm.invalidateLayoutTree();
 
     // required because active element is lost when 'addField' is called.
-    this._renderInitialFocusEnabled();
+    if (this.initialFocusEnabled) {
```

Comment:
Why do we not check the property of the innerForm, like it is done in `Form._renderProperties`? Can these two properties become inconsistent (e.g. by a custom form that modifies this property during its render-phase)?